### DISCUSSION TITLE
Flow condition keeps failing

### DISCUSSION THREAD

**Discussion by keesvanbemmel** [directus/directus#14019] [Feature Requests]
Hi!

I can't seem to figure out how to get a condition working in a flow. Basically upon creation of a "site_contact" item, I want to check what the value is in the "action" field (either contact or subscribe) and mail to different mail addresses.

The condition operation I added has this filter:
```
{
    "$trigger": {
        "action": {
            "_contains": "contact"
        }
    }
}
```

But it always fails based on this error thrown by Joi:
```
'ValidationError: "$trigger.payload" is not allowed. "$trigger.key" is not allowed. "$trigger.collection" is not allowed'
```

Pretty sure my filter condition is messed up, but it's unclear to me what it should be.

Any help is greatly appreciated!

PS perhaps I can't use the field name `action` ?

**Comment by azrikahar** [comment]
The Joi error is not intended and it's currently tracked as an issue in #13855 👍
